npm-shrinkwrap.json

```diff
@@ -321,8 +432,8 @@
     },
     "fxa-auth-db-mysql": {
       "version": "0.59.0",
-      "from": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#70e3ce0ddded7d34876f569eeb97d1a93363c66f",
-      "resolved": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#70e3ce0ddded7d34876f569eeb97d1a93363c66f",
+      "from": "git+https://github.com/eoger/fxa-auth-db-mysql.git#sync-push",
```

Comment:
This is done to make sure tests pass against the other required patch (https://github.com/mozilla/fxa-auth-db-mysql/pull/133)